{"uuid":"c05ac9f3-2724-47a5-92ba-21082ffdd1b9","name":"appium","children":["f9324d02-8a0d-4e79-9bde-d3b1e7c3f2d6","6dc19e6e-b389-4264-8887-08a0db9852d5","96b9c806-d446-4b2e-80f7-18382cbcaea0","85274e52-0f85-4e8f-b0f1-3e551784757d","61fe4007-02fd-4407-87a6-b64dfe6ba0bc"],"befores":[{"name":"setUp","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.net.ConnectException","trace":"java.lang.RuntimeException: java.net.ConnectException\r\n\tat Methods.RequestMethods.registerRequest(RequestMethods.java:33)\r\n\tat Cases.LogInSystem.setUp(LogInSystem.java:34)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:578)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)\r\n\tat org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:644)\r\n\tat org.testng.TestRunner.beforeRun(TestRunner.java:633)\r\n\tat org.testng.TestRunner.run(TestRunner.java:595)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\nCaused by: java.net.ConnectException\r\n\tat java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:846)\r\n\tat java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)\r\n\tat Methods.RequestMethods.registerRequest(RequestMethods.java:31)\r\n\t... 22 more\r\nCaused by: java.net.ConnectException\r\n\tat java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1045)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:224)\r\n\tat java.net.http/jdk.internal.net.http.AsyncSSLConnection.connectAsync(AsyncSSLConnection.java:56)\r\n\tat java.net.http/jdk.internal.net.http.Http2Connection.createAsync(Http2Connection.java:380)\r\n\tat java.net.http/jdk.internal.net.http.Http2ClientImpl.getConnectionFor(Http2ClientImpl.java:128)\r\n\tat java.net.http/jdk.internal.net.http.ExchangeImpl.get(ExchangeImpl.java:94)\r\n\tat java.net.http/jdk.internal.net.http.Exchange.establishExchange(Exchange.java:368)\r\n\tat java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl0(Exchange.java:500)\r\n\tat java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl(Exchange.java:405)\r\n\tat java.net.http/jdk.internal.net.http.Exchange.responseAsync(Exchange.java:397)\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:409)\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsyncImpl$7(MultiExchange.java:450)\r\n\tat java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)\r\n\tat java.base/java.util.concurrent.CompletableFuture.uniHandleStage(CompletableFuture.java:950)\r\n\tat java.base/java.util.concurrent.CompletableFuture.handle(CompletableFuture.java:2372)\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:440)\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsync0$2(MultiExchange.java:342)\r\n\tat java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150)\r\n\tat java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\r\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)\r\n\tat java.net.http/jdk.internal.net.http.HttpClientImpl$DelegatingExecutor.execute(HttpClientImpl.java:172)\r\n\tat java.base/java.util.concurrent.CompletableFuture.completeAsync(CompletableFuture.java:2719)\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.responseAsync(MultiExchange.java:295)\r\n\tat java.net.http/jdk.internal.net.http.HttpClientImpl.sendAsync(HttpClientImpl.java:931)\r\n\tat java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:825)\r\n\t... 24 more\r\nCaused by: java.nio.channels.UnresolvedAddressException\r\n\tat java.base/sun.nio.ch.Net.checkAddress(Net.java:149)\r\n\tat java.base/sun.nio.ch.Net.checkAddress(Net.java:157)\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.checkRemote(SocketChannelImpl.java:842)\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:865)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:208)\r\n\tat java.base/java.security.AccessController.doPrivileged(AccessController.java:569)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:210)\r\n\t... 47 more\r\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1684323381157,"stop":1684323393889}],"afters":[],"start":1684323381132,"stop":1684323394151}